hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/service/TestKeyDeletingService.java

```diff
@@ -670,6 +669,7 @@ public void testSnapshotExclusiveSize() throws Exception {
              iterator = snapshotInfoTable.iterator()) {
       while (iterator.hasNext()) {
         Table.KeyValue<String, SnapshotInfo> snapshotEntry = iterator.next();
+        System.out.println(snapshotEntry.getValue());
```

Comment:
nit: i guess it was miss previously.

Reply:
Yeah, Realized it when raising the new PR. So removed it.

Reply:
Removed the unnecessary `println`.